pandevice/firewall.py

```diff
@@ -63,6 +63,8 @@ class Firewall(PanDevice):
         "device.Vsys",
         "device.VsysResources",
         "device.SystemSettings",
+        "device.PasswordProfile",
+        "device.Administrator",
```

Comment:
These should be added to the `panorama.Panorama` class too.  And templates some day when we support them, but we don't have a Template object yet.